include/posix/time.h

```diff
@@ -10,14 +10,7 @@
 extern "C" {
 #endif
 
-
-#ifndef __timespec_defined
-#define __timespec_defined
-struct timespec {
-	signed int tv_sec;
-	signed int tv_nsec;
-};
-#endif
+#include <time.h>
```

Comment:
Is this maybe including itself because . is in the include search path?

Reply:
Actually the shippable shows `-I../../../../../../include/posix` is in the search path, which is flat out wrong, unless that's how the posix subsystem was designed to be used, in which case there's good reason to step back and rethink it.